examples/chip-tool-darwin/commands/pairing/PairingDelegateBridge.mm

```diff
@@ -0,0 +1,94 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#include "PairingDelegateBridge.h"
+
+using namespace ::chip;
+using namespace ::chip::Controller;
+
+const uint8_t kEndpoint = 0;
+
+CHIPDevice * CHIPGetConnectedDevice(NodeId nodeID)
+{
+    CHIPDeviceController * controller = [CHIPDeviceController sharedController];
+    CHIPDevice * __block connectedDevice;
+    [controller getConnectedDevice:nodeID
+                             queue:dispatch_get_main_queue()
+                 completionHandler:^(CHIPDevice * _Nullable device, NSError * _Nullable error) {
+                     connectedDevice = device;
+                     if (error) {
+                         NSLog(@"Error on getting device: %@", error);
+                     }
+                 }];
+    return connectedDevice;
+}
```

Comment:
You can probably remove this at the moment. Until it is used in cluster calls.

Reply:
Done